SQL/CRUD/CRUDHomework2.sql

```diff
@@ -0,0 +1,14 @@
+UPDATE `agents`
+SET COMMISSION=COMMISSION-.02
+WHERE `agents`.`AGENT_CODE` = (
+	SELECT `customer`.`AGENT_CODE`
+    FROM `customer`
+    GROUP BY `customer`.`AGENT_CODE`
+    HAVING SUM(`customer`.`PAYMENT_AMT`) = (
+		SELECT SUM(`customer`.`PAYMENT_AMT`)
+        FROM `customer`
+        GROUP BY `customer`.`AGENT_CODE`
+        ORDER BY SUM(`customer`.`PAYMENT_AMT`) ASC
+        LIMIT 1
```

Comment:
You should short your sub-query by selecting the `AGENT_CODE` in the same query you are using to order the sum. 
It means you can use only one sub-query to select the `AGENT_CODE`